src/test/regress/expected/multi_alter_table_add_constraints.out

```diff
@@ -488,3 +488,162 @@ SELECT "Constraint", "Definition" FROM table_checks WHERE relid='public.products
+        1
+    LIMIT 
+        1;
+    $$)
+ORDER BY 
+    1;
+          run_command_on_workers          
+------------------------------------------
+ (localhost,57637,t,alter_pk_idx_1450234)
+ (localhost,57638,t,alter_pk_idx_1450234)
+(2 rows)
+
+CREATE SCHEMA sc2;
+CREATE TABLE sc2.alter_add_prim_key(x int, y int);
+CREATE UNIQUE INDEX CONCURRENTLY alter_pk_idx ON sc2.alter_add_prim_key(x);
+SELECT create_distributed_table('sc2.alter_add_prim_key', 'x');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+ALTER TABLE sc2.alter_add_prim_key ADD CONSTRAINT alter_pk_idx PRIMARY KEY USING INDEX alter_pk_idx;
+NOTICE:  using one-phase commit for distributed DDL commands
+HINT:  You can enable two-phase commit for extra safety with: SET citus.multi_shard_commit_protocol TO '2pc'
+SELECT run_command_on_workers($$
+    SELECT  
+        kc.constraint_name 
+    FROM 
+        information_schema.table_constraints tc join information_schema.key_column_usage kc on (kc.table_name = tc.table_name and kc.table_schema = tc.table_schema and kc.constraint_name = tc.constraint_name)
+    WHERE
+        kc.table_schema = 'sc2' and tc.constraint_type = 'PRIMARY KEY' and kc.table_name LIKE 'alter_add_prim_key_%' 
+    ORDER BY
+        1
+    LIMIT 
+        1;
+    $$)
+ORDER BY 
+    1;
+          run_command_on_workers          
+------------------------------------------
+ (localhost,57637,t,alter_pk_idx_1450236)
+ (localhost,57638,t,alter_pk_idx_1450236)
+(2 rows)
+
+CREATE SCHEMA sc3;
+CREATE TABLE sc3.alter_add_prim_key(x int, y int);
+SET search_path TO 'sc3';
+SELECT create_distributed_table('alter_add_prim_key', 'x');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+CREATE UNIQUE INDEX CONCURRENTLY alter_pk_idx ON alter_add_prim_key(x);
+ALTER TABLE alter_add_prim_key ADD CONSTRAINT alter_pk_idx PRIMARY KEY USING INDEX alter_pk_idx;
+SELECT run_command_on_workers($$
+    SELECT  
+        kc.constraint_name 
+    FROM 
+        information_schema.table_constraints tc join information_schema.key_column_usage kc on (kc.table_name = tc.table_name and kc.table_schema = tc.table_schema and kc.constraint_name = tc.constraint_name)
+    WHERE
+        kc.table_schema = 'sc3' and tc.constraint_type = 'PRIMARY KEY' and kc.table_name LIKE 'alter_add_prim_key_%'
+    ORDER BY
+        1
+    LIMIT 
+        1;
+    $$)
+ORDER BY 
+    1;
+          run_command_on_workers          
+------------------------------------------
+ (localhost,57637,t,alter_pk_idx_1450238)
+ (localhost,57638,t,alter_pk_idx_1450238)
+(2 rows)
+
+CREATE SCHEMA sc4;
+CREATE TABLE sc4.alter_add_prim_key(x int);
+INSERT INTO sc4.alter_add_prim_key(x) SELECT generate_series(1,100);
+SET search_path TO 'sc4';
+SELECT create_distributed_table('alter_add_prim_key', 'x');
+NOTICE:  Copying data from local table...
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+CREATE UNIQUE INDEX CONCURRENTLY alter_pk_idx ON alter_add_prim_key(x);
```

Comment:
Let's add a single line comment why we're running `almost` the same test with the above.